pkg/readiness/cert_tracker.go

```diff
@@ -0,0 +1,112 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package readiness
```

Comment:
Do we want to make these two trackers sub-packages of `readiness` to avoid namespace clutter?

Reply:
If we add sub-packages, then all the existing `Tracker` related files will need to move under another sub-package e.g. `Tracker` and the cert tracker related files under `CertTracker`. 

WDYT? @shomron

Reply:
I'm comfortable with some refactoring here.